mParticle-Apptimize/MPKitApptimize.m

```diff
@@ -0,0 +1,133 @@
+//
+//  MPKitApptimize.m
+//
+//  Copyright 2016 mParticle, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "MPKitApptimize.h"
+#import "mParticle.h"
+
+#import "Apptimize/Apptimize.h"
+
+@implementation MPKitApptimize
+
+static NSString *const ALIAS_KEY = @"mparticleAlias";
+static NSString *const CUSTOMER_ID_KEY = @"mparticleCustomerId";
+static NSString *const VIEWED_EVENT_FORMAT = @"Viewed %@ Screen";
+
++ (NSNumber *)kitCode {
+    return @105;
+}
+
++ (void)load {
+    MPKitRegister *kitRegister = [[MPKitRegister alloc] initWithName:@"Apptimize" className:@"MPKitApptimize" startImmediately:YES];
+    [MParticle registerExtension:kitRegister];
+}
+
+#pragma mark - MPKitInstanceProtocol methods
+
+#pragma mark Kit instance and lifecycle
+- (nonnull instancetype)initWithConfiguration:(nonnull NSDictionary *)configuration startImmediately:(BOOL)startImmediately {
+    self = [super init];
+    NSString *appKey = configuration[@"appKey"];
```

Comment:
the casing of `appKey` is different here and on line 62. I would suggest defining a constant and using that in both places.